pom.xml

```diff
@@ -493,6 +494,11 @@
         <artifactId>jets3t</artifactId>
         <version>0.8.1</version>
       </dependency>
+      <dependency>
```

Comment:
double check that the new dependencies are compatible with apache 2.0 license and not introducing any forbidden dependencies as described in https://www.apache.org/legal/resolved.html#category-x

Reply:
New dependencies;
1. [JNA](https://github.com/java-native-access/jna) (`net.java.dev.jna:jna-jpms`): [LGPL and Apache 2.0 dual licensed](https://github.com/java-native-access/jna/blob/master/LICENSE).
2. [Maven Archetype Plugin](https://maven.apache.org/archetype/maven-archetype-plugin/index.html): Apache 2.0 as it's part of the Maven project
3. [Maven plugin for Rust](https://github.com/questdb/rust-maven-plugin)(`org.questdb:rust-maven-plugin`): [Apache 2.0](https://github.com/questdb/rust-maven-plugin/blob/main/LICENSE).